client/a8c-for-agencies/sections/referrals/lib/get-account-status.ts

```diff
@@ -0,0 +1,58 @@
+export const getAccountStatus = (
+	data: {
+		Status: string;
+		IsPayable: boolean;
+		PayableReason: string[];
+	} | null,
+	translate: ( key: string ) => string
+): {
+	statusType: 'success' | 'warning' | 'error';
+	status: string;
+	statusReason?: string;
+} | null => {
+	if ( ! data ) {
+		return null;
+	}
+	const { Status, IsPayable, PayableReason } = data;
+	switch ( Status ) {
+		case 'Active':
+			if ( ! IsPayable ) {
+				return {
+					statusType: 'warning',
+					status: translate( 'Not Payable' ),
+					statusReason: PayableReason?.map( ( reason ) => {
+						if ( reason === 'No PM' ) {
+							return translate( 'Bank details are missing' );
+						}
+						if ( reason === 'Tax' ) {
+							return translate( 'Tax form is missing' );
+						}
```

Comment:
Good job customizing the text for these! All other messages are descriptive enough, but these two seems to be a bit confusing, so it's good that you added an extra explanation.